src/main/java/bg/codexio/springframework/boot/adapters/GraphQLHttpFilterAdapter.java

```diff
@@ -0,0 +1,457 @@
+                    Map.entry(
+                            "_not_in",
+                            FilterOperation.NOT_IN
+                    ),
+                    Map.entry(
+                            "_contains",
+                            FilterOperation.CONTAINS
+                    ),
+                    Map.entry(
+                            "_starts_with",
+                            FilterOperation.BEGINS_WITH
+                    ),
+                    Map.entry(
+                            "_ends_with",
+                            FilterOperation.ENDS_WITH
+                    ),
+                    Map.entry(
+                            "_empty",
+                            FilterOperation.EMPTY
+                    ),
+                    Map.entry(
+                            "_not_empty",
+                            FilterOperation.NOT_EMPTY
+                    ),
+                    Map.entry(
+                            "_begins_with_caseins",
+                            FilterOperation.BEGINS_WITH_CASEINS
+                    ),
+                    Map.entry(
+                            "_ends_with_caseins",
+                            FilterOperation.ENDS_WITH_CASEINS
+                    ),
+                    Map.entry(
+                            "_contains_caseins",
+                            FilterOperation.CONTAINS_CASEINS
+                    )
+            );
+    private final Logger logger =
+            LoggerFactory.getLogger(GraphQLHttpFilterAdapter.class);
+    private final ObjectMapper objectMapper;
+    private final GraphQLComplexFilterAdapter graphQLComplexFilterAdapter;
+
+
+    /**
+     * Constructs a {@code GraphQLHttpFilterAdapter} with the specified
+     * dependencies.
+     *
+     * @param objectMapper                the {@link ObjectMapper} for JSON
+     *                                    processing
+     * @param graphQLComplexFilterAdapter the adapter for handling complex
+     *                                    GraphQL filters
+     */
+    public GraphQLHttpFilterAdapter(
+            ObjectMapper objectMapper,
+            GraphQLComplexFilterAdapter graphQLComplexFilterAdapter
+    ) {
+        this.objectMapper = objectMapper;
+        this.graphQLComplexFilterAdapter = graphQLComplexFilterAdapter;
+    }
+
+    /**
+     * Determines whether the given {@link HttpServletRequest} supports
+     * GraphQL requests.
+     * <p>
+     * This method checks if the request URL contains the "/graphql" path,
+     * which indicates
+     * that the request is related to GraphQL operations.
+     * </p>
+     *
+     * @param req the {@link HttpServletRequest} to evaluate
+     * @return {@code true} if the request URL contains "/graphql", {@code
+     * false} otherwise
+     */
+    @Override
+    public boolean supports(HttpServletRequest req) {
+        return req.getRequestURL()
+                  .toString()
+                  .contains("/graphql");
```

Comment:
This shouldn't be hardcoded, but rather comes from configuration properties. I would also suggest to have in the configuration properties a boolean that denotes whether the request body shall be tested, and if true, to test it.

Also `contains` is not the best operation. It should be a pattern.